journal-letters/modernized/15-chapter-14.adoc

```diff
@@ -118,22 +125,23 @@ or we will not allow any of His messengers and servants to dwell among us,
 but we will imprison them, and put them to death,`' etc.
 Thus the Lord may permit them to do,
 until they have filled up the measure of their iniquities, as the Amorites did;
-and then shall his iron rod be stretched over them,
+and then shall His iron rod be stretched over them,
 by which they shall be broken to pieces like a potter`'s vessel, who have abhorred Him,
 and hated Him without a cause; but in that day will he spare you,
-who have followed Him through great tribulation.-- In the meantime, O! beloved,
+who have followed Him through great tribulation.-- In the meantime, O! Beloved,
 he will try your faith and patience; but be you not therefore troubled,
 for he knows what is good for you,
 in whom he has chosen in these latter days to manifest his power and glory,
 to the families of the earth,
 whose glory and dignity must be stained and brought to nothing.
 
+[.offset]
 +++[+++He then proceeds to give a similar narrative of his labors in Friesland, etc.,
 though somewhat more minute than given at this place in the Journal.
-He afterwards adds;]
+He afterwards adds;+++]+++
 
 The sudden and violent storm which you have had in England,^
-footnote:[See the Histories of England concerning the
+footnote:[See the [.book-title]#Histories of England# concerning the
```

Comment:
You did this perfect. 👍

Reply:
Thanks to the style guide and preview button.... It took me about 17 versions of it to get it to appear correctly! But eventually I hit the nail on the head!